tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "preserveConstEnums": true,
     "strictNullChecks": true,
     "sourceMap": true,
-    "target": "es5"
+    "target": "es5",
+    "importHelpers": true
```

Comment:
This is cool, I didn't know this was a thing. Should this be in our [TS guide](https://github.com/resin-io/process/blob/master/process/TypeScript_Coding_Guide.md) as something we do everywhere? Seems like a good idea.